iambic/core/utils.py

```diff
@@ -135,6 +135,9 @@ async def file_regex_search(file_path: str, re_pattern: str) -> Union[str, None]
 
 
 async def gather_templates(repo_dir: str, template_type: str = None) -> list[str]:
+    if repo_dir.endswith("/") and repo_dir != "//":
```

Comment:
Consider using pathlib to resolve the path, I think pathlib has a bunch of built-ins. The reason for the suggestion is that we also have to account for Windows UNCs, which differ.